godot-macros/src/class/data_models/inherent_impl.rs

```diff
@@ -89,6 +93,19 @@ pub fn transform_inherent_impl(
     #[cfg(not(all(feature = "register-docs", since_api = "4.3")))]
     let docs = quote! {};
 
+    // This is the container struct that holds the names of all registered #[func]s.
+    // (The struct is declared by the macro derive_godot_class.)
+    let class_functions_name = format_func_name_struct_name(&class_name);
```

Comment:
tbh it would be good to add comment over this section that specify what is going on; something along the lines of `// handle class renaming` – it took me a while to realize what is going on :sweat_smile: